tests/src/JIT/Regression/JitBlue/GitHub_7147/GitHub_7147.csproj

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@echesakovMSFT has changed the test format, could you please replace this one (in the old format) with one of the existing csproj files?